frontend/src/app/event-list/event-list.component.ts

```diff
@@ -1,54 +1,31 @@
 import { Component, OnInit } from '@angular/core';
-import { BehaviorSubject } from 'rxjs';
+import { Observable } from 'rxjs';
+import { GeoEventsService } from '../modules/data/services/geo-events.service';
+import { IGeoEvent } from '~model/interfaces/geo-event.interface';
 import datetimeUtils from '../utils/datetime';
 
-// TODO: replace by GeoEventService, blocked by #9
-const createMockGeoEvent = (props: any) => ({
-  id: '0',
-  center: {
-    latitude: 48.858093,
-    longitude: 2.294694,
-  },
-  title: 'Start of construction of the Eiffel Tower',
-  timestamp: new Date(2022, 1, 15, 20, 50),
-  ...props
-})
-
-// TODO: replace by GeoEventService, blocked by #9
-const today = new Date();
-const geoEventMocks = new BehaviorSubject([
-  createMockGeoEvent({ id: 0, address: '1 Street, City Name, Country' }),
-  createMockGeoEvent({ id: 1, timestamp: new Date(today.getTime() - 1000 * 24) }),
-  createMockGeoEvent({ id: 2, timestamp: new Date(today.getTime() - 1000 * 60 * 1) }),
-  createMockGeoEvent({ id: 3, timestamp: new Date(today.getTime() - 1000 * 60 * 50) }),
-  createMockGeoEvent({ id: 4, timestamp: new Date(today.getTime() - 1000 * 60 * 60) }),
-  createMockGeoEvent({ id: 5, timestamp: new Date(today.getTime() - 1000 * 60 * 60 * 23) }),
-  createMockGeoEvent({ id: 6, timestamp: new Date(today.getTime() - 1000 * 60 * 60 * 24 * 10) }),
-  createMockGeoEvent({ id: 7, title: 'some stupidly long title wow its so long and useless why would ppl fill in something this long and then we have to deal with them ahhhhhhhhhhhhh'}),
-]);
-
 @Component({
   selector: 'app-event-list',
   templateUrl: './event-list.component.html',
   styleUrls: ['./event-list.component.scss']
 })
 export class EventListComponent implements OnInit {
 
-  constructor() { }
+  public geoEvents$: Observable<IGeoEvent[]> | null = null;
+
+  constructor(private geoEventsService: GeoEventsService) { }
 
   ngOnInit(): void {
+    this.geoEvents$ = this.geoEventsService.getAll();
   }
 
   getTimePassed = (timestamp: Date) => {
     return datetimeUtils.getTimePassed(timestamp)
   }
 
-  getAddress = (geoEvent: any) => {
-    const { address, center } = geoEvent;
+  getAddress = ({ address, coordinates }: IGeoEvent) => {
     if (address) return address;
-    return `${center.longitude} ${center.latitude}`
+    return `${coordinates.longitude} ${coordinates.latitude}`
```

Comment:
I don't like this kind of writing so much. This is clearly an if/else statement and it takes longer to understand writing it in this shortened form.

I'd say:
```suggestion
return address ?? `${coordinates.longitude} ${coordinates.latitude}`
```
or a conditional statement, but there you repeat the word 'address'

Reply:
Is this better?
```js
getAddress = ({ address, coordinates }: IGeoEvent) => {
  return address || `${coordinates.longitude} ${coordinates.latitude}`;
}
```